airflow/www/templates/airflow/conn_create.html

```diff
@@ -21,6 +21,10 @@
 
 {% block tail %}
   {{ super() }}
+  <script>
+    // eslint-disable-next-line no-unused-vars
+    const restApiEnabled = '{{ rest_api_enabled }}' === 'True';
```

Comment:
See how we use `import getMetaValue from './meta_value';` in other js files. We should do the same here.